brian_cheung/week_two_challenges/kanye_records.md

```diff
@@ -19,49 +19,65 @@ kanyeRecords = [
 1. Print the name of every Kanye record.
 
 ```ruby
-# write your code here
+kanyeRecords.each do |hash_iterate| puts hash_iterate[:title] end
 ```
```

Comment:
Nothing is technically stopping you from putting this code on one line, but generally if you're using `do/end` then the block should be on three lines.

You could reformat to use the `item.each { |x| <block> }` syntax - that's generally acceptable to keep on one line.

Reply:
oh, thats how you do it with one line..